tox.ini

```diff
@@ -53,7 +53,7 @@ commands =
     python run_pylint.py
 deps =
     pep8
-    pylint
+    -ehg+https://bitbucket.org/logilab/pylint#egg=pylint
```

Comment:
Ah, OK -- the `-e` is a pip-ism, not related to the URL itself:  it tells pip to install it in "development mode".  I was worried it might've been a typo for `egg+https` or some such.